src/content/release-notes.mdx

```diff
@@ -5,6 +5,44 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing [Cloud Credentials](admin/cloud-credentials/index.mdx), a central location to manage your cloud provider credentials for `deploy`, `destroy`, and `test` remote operations  <!-- 8550 -->
+- Adding support for [resource quotas at the individual Namespace level](core/namespaces.mdx#configure-namespace-quotas). This allows administrators to set the maximum resources that can be used per Namespace
+- Enabling [Okteto Insights Dashboards](core/okteto-insights-dashboards.mdx) for all SaaS and BYOC users of Okteto <!-- 8627 -->
+- Added CLI, Chart, and Okteto App version to our API headers  <!-- 8545 -->
```

Comment:
What does "our" refer to more specifically? How does this affect/help customers? Asking because looking at referenced PR and parent issue, it appears to be more for internal usage than "new feature" that any customer would use, isn't it?

Reply:
Oddly enough we've received a few customer requests to have this version data so that our customer's platform team can have more information when debugging developer issues.
Good call out though on "our" I'll figure out a way to reword this to be more clear and show the value prop